README.md

```diff
@@ -60,5 +60,70 @@ To register a device, run as root from LmP device:
 (i.e. `sudo DEVICE_API=http://10.0.2.2:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]`)
 
 # Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
+
+In scenarios where you'd like to test the API call without involving a physical device or emulation:
```

Comment:
The below paragraph is enough for the user to make the decision for their testing needs.
So, please, remove this sentence.